pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -725,23 +732,8 @@ func (r *ReconcileVirtualMachineImport) createDataVolume(provider provider.Provi
 	err = r.client.Create(context.TODO(), &dv)
 	if err != nil {
 		message := fmt.Sprintf("Data volume %s/%s creation failed: %s", dv.Namespace, dv.Name, err)
-		// Update event:
-		r.recorder.Event(instance, corev1.EventTypeWarning, EventDVCreationFailed, message)
-
-		// Update condition to failed:
-		succeededCond := conditions.NewSucceededCondition(string(v2vv1alpha1.DataVolumeCreationFailed), message, corev1.ConditionFalse)
```

Comment:
any reason why we do not need this condition any more?

Reply:
It is added on the caller level @ line 356.

Reply:
OK